test/unit/test_transaction.cpp

```diff
@@ -266,4 +266,42 @@ TEST_F(test_transaction, trivial_types_dup_iterate_by_key)
     ASSERT_TRUE(result);
```

Comment:
I think there is a small bug in `trivial_types_dup_iterate_by_key`, `iterate_by_key` call should be used instead. I will look into that as a different PR.